packages/odyssey-react/src/components/FieldGroup/FieldGroup.module.scss

```diff
@@ -13,12 +13,12 @@
 .root {
   max-width: $max-line-length;
   margin-block-start: 0;
-  margin-block-end: $spacing-m;
+  margin-block-end: var(--SpaceEmM);
   padding-inline: 0;
   padding-block-start: 0;
-  padding-block-end: $spacing-m;
+  padding-block-end: var(--SpaceEmM);
```

Comment:
You might want to split these into `MarginBlockEnd` and `PaddingBlockEnd` or similar. We keep them uniform, but I don't know if that's required. So far, we've been splitting up most local values and maximizing customization.